util/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './utils';
+export * as FileSystem from './fileSystem';
+export { Constants } from './constants';
+export { Errors } from './errors';
+export * from './fattr';
```

Comment:
all fattr functions should stay in FileAttr object, I believe currently they are at top level